var/spack/repos/builtin/packages/r-functional/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class RFunctional(RPackage):
+    """Curry, Compose, and other higher-order functions
+
+    Curry, Compose, and other higher-order functions"""
+
```

Comment:
Mind adding the `homepage` property?

Reply:
Somewhat unusually, it doesn't have a homepage or associated GitHub repo. Should I just set the homepage to its [CRAN homepage](https://cran.r-project.org/web/packages/functional/index.html)? I'm not sure what the convention is for packages like that.

Reply:
Ick.  It does provide a link to the reference manual but not sure it's worth adding so I'll leave that up to you.